src/main/java/hudson/plugins/git/browser/GitLab.java

```diff
@@ -134,6 +135,7 @@ public URL getFileLink(Path path) throws IOException {
     }
 
     @Extension
+    @Symbol("gitLab")
```

Comment:
Since the GitLab plugins [already provide this symbol](https://github.com/jenkinsci/gitlab-plugin/blob/b7c5c92352f80e88cc9c77274760cecbf073502b/src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java#L566), we should use a different symbol
```suggestion
    @Symbol("gitLabBrowser")
```

Reply:
Similarly, should we as a policy include `Repository` to these symbols?

It would make them long, but the added length would make the symbol names consistent. E.g., if one was using Fisheye, this symbol could become (`FisheyeRepositoryBrowser`)[https://github.com/jenkinsci/git-plugin/compare/master...MarkEWaite:add-symbols#diff-a60a017f74a135172edb2f91d283a564R70]. If someone were writing a Pipeline `checkout` for example, it might be more obvious if they all followed the same pattern, rather than having some include `browser` and others not.

Reply:
I like the proposal from @dwnusbaum that we try for short symbols and test for surprises.  No `Repository` in the symbol and no `Browser` in the symbol.

Reply:
This is a good one to test "as is" (gitLab) since that is the same symbol name provided by the GitLab plugin.  I have the GitLab plugin installed and the git plugin installed, so I should be able to check that the symbols do not collide or conflict.